docs/intro-to-dashboards-and-apps/connecting-etl-to-dashboard-with-voila.md

```diff
@@ -6,9 +6,178 @@ jupyter:
+
+[Install](https://github.com/voila-dashboards/voila) `Voila` with the following command:
+
+```bash
+pip install voila
+```
+
+## Questions to Answer in the Dashboard
+
+The interactive dashboard contains 4 tables and 5 plots, created using `JupySQL`'s `ggplot` API, `seaborn`, and `ipywidgets`. It answers the following questions:
+
+1. How do yearly manufacturing trends of fuel-only, electric, and hybrid cars compare?
+2. How are fuel consumption and $CO2$ emissions distributed for all types of cars?
+3. What is the relationship between charging time and travel range for electric vehicles by car size and model year?
+4. How are $CO_2$ emissions distributed by vehicle type (fuel-only, electric, and hybrid) and fuel type (gasoline, diesel, ethanol, natural gas, and electricty)?
+5. Which US fuel-only and hybrid car manufacturers emit the least $CO_2$ and how does this differ by transmission type?
+
+## ETL and Voila
+
+Recall the [ETL Pipeline](https://ploomber-sql.readthedocs.io/en/latest/packaging-your-sql-project/intro-to-etl-pipelines-with-python-and-sql.html) walkthrough in the previous module. The Ploomber pipeline executes the `datadownload.py` ETL [script](https://github.com/ploomber/sql/blob/main/pipeline/src/datadownload.py) as well as the EDA Jupyter [notebook](https://github.com/ploomber/sql/blob/main/pipeline/src/eda-pipeline.ipynb) with selected queries. It then stores the tables in an in-memory DuckDB database. We can connect our dashboard to the DuckDB instance and generate queries for our visualizations. This is done in the following code cell:
+
+```python
+%load_ext sql
+
+%sql duckdb:///../data/database/car_data.duckdb
+```
+
+The `%sql` magic command allows us to connect to the database and query the data. The `duckdb:///` prefix specifies the database type and location. The `../data/database/car_data.duckdb` path specifies the location of the database relative to the notebook. The `%sql` magic command is used in the following code cells to create CTE's for generating visualizations.
+
+<b>Note:</b> The `duckdb:///` prefix is not required if the database is in the same directory as the notebook.
+
+The pipeline process entailed above can be better understood with the following diagram:
+
+![ETL Pipeline](../packaging-your-sql-project/dashboard-comp.jpg)
+
+## Dashboard Structure
+
+### Introduction and Tables
+
+The dashboard, firstly, needs to have a relevant title and description for the user to understand what the dashboard is about. The date the fuel emissions data was last updated is also displayed because the data is updated monthly and, accordingly, the tables and visualizations may have novel insights since the last update. Next, we display the `all_vehicles` table, integrated with `ipywidgets` and outputted from our ETL Pipeline, to allow the user interact with different columns in each table. The user can, hence, begin the EDA process by filtering columns to find interesting patterns and relationships.
```

Comment:
It would be good to embed the table portion of the voila app so that people can see it

Reply:
Using the `itable` package to output 5 columns (the ones in the widget) without column names overlapping